### PR TITLE
[Edge] refactoring edge cmd @open sesame 07/01 10:33

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Refactoring to send/receive edge command and buffer.
- define command info struct to handle command, id, and memories.
- util functions to init, clear edge command.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c [hunk 1]
```diff
@@ -68,7 +75,7 @@ typedef struct
 } nns_edge_conn_data_s;
 
 /**
- * @brief Structures for thread data of tensor query messge handling.
+ * @brief Structures for thread data of messge handling.
```
Comment (anyj0527) [inline gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c:78]: `messge` -> `message`

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c [hunk 2]
```diff
@@ -238,7 +435,7 @@ _nns_edge_get_saddr (const char *ip, int port,
     results = g_resolver_lookup_by_name (resolver, ip, cancellable, &err);
     if (!results) {
       if (g_error_matches (err, G_IO_ERROR, G_IO_ERROR_CANCELLED)) {
-        nns_edge_logd ("gst_tensor_query_socket_new: Cancelled name resolval");
+        nns_edge_logd ("_nns_edge_get_saddr: Cancelled name resolval");
```
Comment (anyj0527) [inline gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c:438]: `resolval` -> `resolver`

**Review by anyj0527 — APPROVED** [review]
LGTM✔ with trivial typos.

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c]: updated

**Pull request merged**